test/src/main/java/org/apache/accumulo/test/functional/DurabilityIT.java

```diff
@@ -165,10 +165,10 @@ public void testMetaDurability() throws Exception {
     try (AccumuloClient c = Accumulo.newClient().from(getClientProperties()).build()) {
       String tableName = getUniqueNames(1)[0];
       c.instanceOperations().setProperty(Property.TABLE_DURABILITY.getKey(), "none");
-      Map<String,String> props = map(c.tableOperations().getProperties(MetadataTable.NAME));
```

Comment:
Not sure if the map method is still needed after this change. If it isn't needed, it should be deleted.